validator/schema/upsun.json

```diff
@@ -160,17 +183,76 @@
           },
           "runtime": {
             "type": "object",
             "title": "Runtime-specific configuration.",
-            "default": {}
+            "description": "Customizations to your PHP or Lisp runtime.\\nContains a runtime dictionary:\\nhttps://docs.upsun.com/create-apps/app-reference/single-runtime-image.html#runtime",
+            "properties": {
+              "extensions": {
+                "type": "array",
+                "title": "PHP extensions to enable.",
+                "description": "More information:\\nhttps://docs.upsun.com/create-apps/app-reference/single-runtime-image.html#extensions",
+                "default": []
+              },
+              "disabled_extensions": {
+                "type": "array",
+                "title": "PHP extensions to disable.",
+                "description": "More information:\\nhttps://docs.upsun.com/create-apps/app-reference/single-runtime-image.html#extensions",
+                "default": []
+              },
+              "request_terminate_timeout": {
+                "type": "integer",
+                "title": "PHP timeout to terminate requests.",
+                "description": "The timeout in seconds for serving a single request after which the PHP-FPM worker process is killed."
+              },
+              "sizing_hints": {
+                "type": "object",
+                "title": "A sizing hints definition.",
+                "description": "The assumptions for setting the number of workers in your PHP-FPM runtime.\\nMore information:\\nhttps://docs.upsun.com/create-apps/app-reference/single-runtime-image.html#sizing-hints",
+                "properties": {
+                  "request_memory": {
+                    "type": "integer",
+                    "title": "The average memory consumed per request in MB.",
+                    "description": "Minimum to 10",
+                    "default": 45,
+                    "minimum": 10
+                  },
+                  "reserved_memory": {
+                    "type": "integer",
+                    "title": "The amount of memory reserved in MB.",
+                    "description": "Minimum to 70",
+                    "default": 70,
+                    "minimum": 70
+                  }
+                },
+                "additionalProperties": false
+              },
+              "xdebug": {
+                "type": "object",
+                "title": "An Xdebug definition",
+                "description": "The setting to turn on Xdebug.\\nMore information:\\nhttps://docs.upsun.com/languages/php/xdebug.html",
+                "properties": {
+                  "idekey": {
+                    "type": "string",
+                    "title": "Your Xdebug key."
+                  }
+                }
+              },
+              "quicklisp": {
```

Comment:
Lisp image will be removed cf https://github.com/platformsh/platformsh-docs/pull/4455, lets remove this entirely?

Reply:
i add a deprecationMessage instead, linking to the devCenter blogpost about Lisp
```
                "deprecationMessage": "Lisp image no longer exists, please see\\nhttps://devcenter.upsun.com/posts/deploying-with-lisp/\\n for more information"
```
Is this ok?

Reply:
Yes sounds good we can always remove it when we have it deleted